ledger/common/convert/convert.go

```diff
@@ -18,6 +20,18 @@ const (
 // UnexpectedLedgerKeyFormat is returned when a ledger key is not in the expected format
 var UnexpectedLedgerKeyFormat = fmt.Errorf("unexpected ledger key format")
 
+// AddressToRegisterOwner converts 8-byte address to register owner.
+// If given address is ZeroAddress, register owner is "" (global register).
+func AddressToRegisterOwner(address common.Address) string {
```

Comment:
Maybe this can be moved to `model/flow/ledger.go`, and used in `NewRegisterID` there?